src/test/java/mate/academy/bookstore/BookstoreApplicationTests.java

```diff
@@ -1,13 +1,8 @@
 package mate.academy.bookstore;
 
-import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 class BookstoreApplicationTests {
 
-    @Test
-    void contextLoads() {
-    }
-
 }
```

Comment:
Why do I need it?

Reply:
As Bogdan told, this method runs the whole application for testing.
Empty contextLoads() is a test to verify if the application is able to load Spring context successfully or not.

Reply:
Firstly i don't have any of tests yet. Secondly, if app can't load spring context, it can't work at all, and i don't think that. And at the end it starts. 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] 
[INFO] Results:
[INFO]
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO]

Reply:
I was just interested in that if your app worked with that method :)

Reply:
I guess that you didn't add jwt.expiration and jwt.secret to application.properties in test folder (I can't find any commits with this changes), therefore you faced the issue with contextLoads test method